src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void drawFigure() {
```

Comment:
The method name 'drawFigure' suggests a specific action. Consider renaming it to simply 'draw' to align with the principle of using nouns for classes/interfaces and verbs for methods. Additionally, the method should be abstract if it's meant to be implemented by subclasses, as the drawing behavior will likely vary between different figures.